EpiAware/src/EpiInfModels/EpiInfModels.jl

```diff
@@ -6,10 +6,14 @@ module EpiInfModels
 using ..EpiAwareBase
 using ..EpiAwareUtils
 
-using Turing, Distributions, DocStringExtensions, LinearAlgebra, LogExpFunctions
+using Turing, Distributions, DocStringExtensions, LinearAlgebra, LogExpFunctions,
+      OrdinaryDiffEq
 
 #Export models
-export EpiData, DirectInfections, ExpGrowthRate, Renewal
+export EpiData, DirectInfections, ExpGrowthRate, Renewal, InfectionODEProcess
```

Comment:
very clunky have ODEParams but InfeectionODEProcess

I think ODEParams and ODEProcess seems reasonable. 

Small detail by the current export comments (models and parameters) aren't really correct. I think ODEParams and EpiData should maybe go together as parameter helpers and everything is a process model?